src/plugins/visualizations/server/migrations/visualization_saved_object_migrations.ts

```diff
@@ -1007,6 +1075,7 @@ export const visualizationSavedObjectTypeMigrations = {
     removeDefaultIndexPatternAndTimeFieldFromTSVBModel
   ),
   '7.13.1': flow(
+    addEmptyValueRuleForSavedObjectsWithLessAndGreaterThenZeroRules,
```

Comment:
We don't want this for 7.13.1
We want this only for 7.14

Moreover the migration logic has a little bit changed in order to take under consideration the by value visualizations (the panels that are created from the dashboard and not saved to the library). So take a look on what we do for 7.13.1 and apply it for 7.14.  Do not forget the `visualize_embeddable_factory`!

Reply:
Ok, thank you.